pkg/api/server/idletracker/idletracker.go

```diff
@@ -0,0 +1,74 @@
+package idletracker
```

Comment:
New package was required to break dependency cycle. I think it makes sense to have this tracker in isolated package